breakout_3_Observability-with-AI/breakout_3_exercise.md

```diff
@@ -0,0 +1,50 @@
+## Part 3 - Observability with AI
+#### How to find the same root cause with your friendly neighborhood Grafana Assistant
+For this exercise you will work through troubleshooting scenarios from the previous sections, however this time using AI tools, specifically [Grafana Assistant](https://grafana.com/docs/grafana-cloud/machine-learning/assistant/), to find the solution.
+
+### Instructions
+You will use the __same environment as part 1__ for this exercise. Go to the URL in the slides that references to the part 1 environment where you will have access to the Grafana Cloud Stack with:
+1. Frontend O11y
+1. App O11y
+1. K8s Monitoring
+
+Navigate to the indicated app in each section, then click on the Grafana Assistant button in the upper right hand corner to get started. The same prompt window can and should be used for a section for context and question continuity.
+![grafana-assistant](/images/breakout_3/0.1-grafana-assistant.png)
+
+You will have 10 minutes to work through the questions, after which we will regroup to work through everything together for the final 10 minutes.
```

Comment:
It takes wayyyyyyy longer than this, it is taking at least 30 minutes and most of that is waiting for responses

Reply:
Updated, will do core and extra credit questions